docker/images/pinot-base/pinot-base-runtime/ms-openjdk.dockerfile

```diff
@@ -20,12 +20,18 @@
 ARG JAVA_VERSION=11
 ARG JDK_IMAGE=mcr.microsoft.com/openjdk/jdk
 
-FROM ${JDK_IMAGE}:${JAVA_VERSION}-ubuntu
+FROM ${JDK_IMAGE}:${JAVA_VERSION}-ubuntu as builder
+
+FROM ubuntu:24.10
+ENV LANG=en_US.UTF-8
+ENV JAVA_HOME=/usr/lib/jvm/msopenjdk-11
```

Comment:
this directory might change based on `JAVA_VERSION` right?

Reply:
Yes, we can make it configurable based on `JAVA_VERSION`.  Let me make the change.